cypress/e2e/flightsearchertest.cy.js

```diff
@@ -0,0 +1,28 @@
+/// <reference types="cypress"/>
+
+describe("Tests for Flight Search", () => {
+    beforeEach( () => {
+        cy.visit('https://www.ryanair.com/')
+
+        cy.get('.cookie-popup-with-overlay__button').click()
+    })
+
+    it('Valid Flight Search Test', () => {
+        cy.selectDeparture();
```

Comment:
remember we were talking about various layers of testing? From the perspective of what you know about our strategies, how would you structure your tests?